docs/src/installing-deis/configuring-object-storage.md

```diff
@@ -0,0 +1,26 @@
+# Configure Object Storage
+
+A variety of Deis components rely on an object storage system to do their work. These components are:
+
+- [workflow](https://github.com/deis/workflow)
+- [builder](https://github.com/deis/builder)
+- [slugbuilder](https://github.com/deis/slugbuilder)
+- [slugrunner](https://github.com/deis/slugrunner)
+- [registry](https://github.com/deis/registry)
+
+These components are built flexibly, so they can work out of the box with almost any system that is compatible with the [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/APIRest.html).
+
+# Minio
+
+Additionally, Deis ships with a [Minio](http://minio.io) [component](https://github.com/deis/minio). This component runs as a Kubernetes service, and the components listed above are configured to automatically look for that service and use it as object storage if it's available.
+
+# Telling Deis What to Use
```

Comment:
forgot about this TODO, I'll fill it in now that I have the lookup order. thanks @carmstrong